lambdas/data-migration2/src/executions.ts

```diff
@@ -79,6 +82,8 @@ export const migrateExecutions = async (
     failed: 0,
     skipped: 0,
   };
+  const errorFile = [];
```

Comment:
Nit: errorMessages or errors?    It's not really a 'file' or filestream object, so much as an array of errorMessages.